tui/actions.go

```diff
@@ -112,30 +113,56 @@ func (ui *UI) AnalyzePath(path string, parentDir *analyze.Dir) error {
 	return nil
 }
 
-func (ui *UI) deleteSelected() {
+func (ui *UI) deleteSelected(isEmpty bool) {
 	row, column := ui.table.GetSelection()
-	selectedFile := ui.table.GetCell(row, column).GetReference().(analyze.Item)
-
-	modal := tview.NewModal().SetText("Deleting " + selectedFile.GetName() + "...")
-	ui.pages.AddPage("deleting", modal, true, true)
+	selectedItem := ui.table.GetCell(row, column).GetReference().(analyze.Item)
 
-	currentDir := ui.currentDir
+	var action, acting string
+	if isEmpty {
+		action = "empty "
+		acting = "emptying"
+	} else {
+		action = "delete "
+		acting = "deleting"
+	}
+	modal := tview.NewModal().SetText(strings.Title(acting) + " " + selectedItem.GetName() + "...")
+	ui.pages.AddPage(acting, modal, true, true)
+
+	var currentDir *analyze.Dir
+	var currentItems []analyze.Item
```

Comment:
What is the meaning of `current` here? Wouldn't `itemsToTruncate` be better?

Reply:
Basically, `current` means `ui.current`. `itemsToTruncate` or `itemsToDelete` would be better.